src/components/filterbox.pcss

```diff
@@ -28,17 +42,23 @@
 
   .search {
     background: none;
-    border: 0;
-    border-top: 1px solid rgb(201, 201, 201);
+    border: 1px solid #d9d9d9;
     box-sizing: border-box;
     display: inline-block;
+    font-family: 'Source sans pro', sans-serif;
```

Comment:
Could we move this declaration to `index.pcss` and have it for `input, textarea` instead?

Reply:
sounds good.